components/logWorkout/LogHeader.jsx

```diff
@@ -17,16 +18,26 @@ const StyledText = styled.Text`
   text-align: center;
 `;
 
-export default () => (
+const LogHeader = ({ type }) => (
   <Container>
     <StyledText style={{ width: 30 }}>Set</StyledText>
 
     <StyledText style={{ width: 70 }}>Previous</StyledText>
 
     <StyledText style={{ width: 70 }}>lbs</StyledText>
 
-    <StyledText style={{ width: 70 }}>Reps</StyledText>
+    {type === 'REPS' ? <StyledText style={{ width: 70 }}>Reps</StyledText> : <StyledText style={{ width: 70 }}>Secs</StyledText>}
```

Comment:
<StyledText style={{ width: 70 }}>{type == 'REPS' ? 'Reps' : 'Secs'}</StyledText>

This does the same thing but it's a little more organized.

Reply:
Yep, good catch